website/src/css/custom.css

```diff
@@ -107,3 +107,65 @@ svg.lightToggleIcon_node_modules-\@docusaurus-theme-classic-lib-next-theme-Color
 li.footer__item a svg {
   display: inline;
 }
+
+/* mo changes 20 oct 2022 */
+
+.alert {
+  max-width: 1500px;
+  margin: 2em auto;
+}
+
+html[data-theme='dark'] .navbar {
+  background-color: #1b1b1d !important;
```

Comment:
you should use  --ifm-background-color: #1b1b1d;

Reply:
got it!